fe1-web/src/__tests__/utils/TestUtils.ts

```diff
@@ -35,23 +35,25 @@ export const org = new PublicKey(mockPublicKey);
 // MOCK LAO
 export const mockLaoName = 'MyLao';
 export const mockLaoCreationTime = new Timestamp(1600000000);
-export const mockLaoIdHash: Hash = Hash.fromStringArray(
+export const mockLaoId: Hash = Hash.fromStringArray(
```

Comment:
Given that now "ids" should be of type `Hash`, I updated the name of the variables to reflect this